modules/tinymce/src/core/main/ts/api/EventTypes.ts

```diff
@@ -45,11 +45,11 @@ export type PlaceholderToggleEvent = { state: boolean };
 
 export type LoadErrorEvent = { message: string };
 
-export interface EditorEventMap extends NativeEventMap {
+export interface EditorEventMap extends Omit<NativeEventMap, 'blur' | 'focus'> {
   'activate': { relatedTarget: Editor };
   'deactivate': { relatedTarget: Editor };
-  'focus': FocusEvent & { blurredEditor?: Editor };
-  'blur': FocusEvent & { focusedEditor?: Editor };
+  'focus': { blurredEditor?: Editor };
```

Comment:
I believe we decided these shouldn't be optional now? It was only because of the fact it conflicted with the native types they were previously optional.

Reply:
You've also missed removing the `FocusEvent` import in this file.